VAMobile/src/screens/HomeScreen/ProfileScreen/SettingsScreen/NotificationsSettingsScreen/NotificationsSettingsScreen.tsx

```diff
@@ -107,14 +103,19 @@ const NotificationsSettingsScreen: FC<NotificationsSettingsScreenProps> = ({ nav
   return (
     <FeatureLandingTemplate backLabel={t('settings.title')} backLabelOnPress={navigation.goBack} title={t('notifications.title')}>
       <Box mb={contentMarginBottom}>
-        {!systemNotificationsOn && alert()}
-        <TextView variant={'MobileBodyBold'} accessibilityRole={'header'} mx={gutter}>
-          {t('notifications.settings.personalize.heading')}
-        </TextView>
-        <TextView variant={'MobileBody'} accessibilityRole={'header'} mx={gutter} mt={condensedMarginBetween}>
-          {personalizeText}
-        </TextView>
-        {preferenceList()}
+        {!systemNotificationsOn ? (
```

Comment:
Consider switching the order so the normal case (on) is first, and the off case is second. Minor but I think it's easier to read

Reply:
Good point, I agree. Just made that change